### ISSUE TITLE
2.2

### ISSUE THREAD

**Issue by JustSpaceisded** [absolllute/Mega-Hack-Pro-Future#1205]
how will megahack work in 2.2

**Comment by NewAlexGamer** [comment]
it won't, we'll need to wait a bit for absolute to rewrite what i think is the entirety of megahack because as he stated, every hack needs to be re-written

**Comment by Brittank88** [comment]
I believe the plan is to write MHv8 via Geode, which may streamline that process, however I assume there will still be a wait time, unless it's possible to start in advance using the versions of 2.2 we do have access to?

**Comment by Brittank88** [comment]
(Maybe Absolute will want to speak on this himself)

**Comment by Silas123Go** [comment]
i wonder though, how long would it take to rewrite everything? if you dont have an answer then thats fine tho, im just curious for a rough estimate

**Comment by ThePL19** [comment]
Will I have to repurchase megahack when MHv8 comes out? i just bought MHv7.

**Comment by Silas123Go** [comment]
no, after you purchase a megahack version thats v6 or above you get all other copies tor free

**Comment by PooPooAss527** [comment]
WHEN WILL IT BE FIXED I REALLY WANT IT TO BE FIXED PLEASE

**Comment by PooPooAss527** [comment]
I WANT MORE INFORMATION I WANNA USE MEGAHACK IN 2.2 LEVELS

**Comment by Vortalieto** [comment]
ME TOOOO

**Comment by ThePL19** [comment]
So mega hack v8 had came out. I don't think I need to repurchase MH for Android since MHv8 is the same price as MHv7 that also comes with MH for Android ($4.99), right?
